### DISCUSSION TITLE
Divider responsive orientation

### DISCUSSION THREAD

**Discussion by davidkhierl** [chakra-ui/chakra-ui#3205] [Q&A] [answered]
`<Divider orientation={{ sm: 'vertical', md: 'horizontal' }}  />` is orientation cant be use with responsive breakpoints?

**Comment by chr-ge** [comment, marked as answer]
It should work with `useBreakpointValue`
```jsx
const orientation = useBreakpointValue({ base: "vertical", md: "horizontal" })

<Divider orientation={orientation } />
```

**Reply by davidkhierl** [reply]
I tried this approach but it still not accepting the value, I end up doing this one instead.
```javascript
const [isLargerThan768] = useMediaQuery('(min-width: 768px)');

<Divider orientation={isLargerThan768 ? 'vertical' : 'horizontal'}  />
```

saw this in progress yet to be included
https://github.com/chakra-ui/chakra-ui/pull/3077

**Reply by rochagbr** [reply]
In my case I neded to put it in this way to work in typescript:
```ts
const orientation = useBreakpointValue<"horizontal" | "vertical">({
    base: "horizontal",
    "2xl": "vertical",
  });
```
